bedrock/firefox/tests/test_views.py

```diff
@@ -634,6 +634,39 @@ def test_firefox_home(self, render_mock):
         assert template == ["firefox/home/index-master.html"]
 
 
+class TestFirefoxGA(TestCase):
+    def assert_ga_attr(self, response):
+        cta_types = ["link", "button"]
+        doc = pq(response.content)
+        links = doc(".mzp-c-button")
+        for link in links.items():
+            cta_data = link.attr("data-cta-type")
+            cta_link = link.attr("data-link-type")
+            if cta_data:
+                contain_cta = any(cta_data in list for list in cta_types)
+                assert contain_cta or "fxa-" in cta_data
+            elif cta_link:
+                # some links with the data-link-type use button instead of download which is what is described
+                # in the docs, so not sure if we should allow this or not
+                cta_link_types = ["download", "button", "link"]
+                assert cta_link in cta_link_types
+            else:
+                print(f"{link} does not contain attr cta-type or link-type")
+                assert False
```

Comment:
This can also be written as `assert False, f"{link} does not contain attr cta-type or link-type"`